### DISCUSSION TITLE
Handling of file uploads with Portman

### DISCUSSION THREAD

**Discussion by danielkocot** [apideck-libraries/portman#439] [Ideas]
How can we handle file uploads with Portman? In particular with content-type binary. I can't anything in the documentation about that.

**Comment by wernermorgenstern** [comment]
I would like to know this too

**Comment by thim81** [comment]
@Lars-Brebels-92 shared his Portman config on how to set a file upload:
https://github.com/apideck-libraries/portman/issues/387#issuecomment-1702471592
